### PR TITLE
fix(ui5-bar): align subheader style

### PR DESCRIPTION
**Pull request by tsanislavgatev**
fixes: https://github.com/SAP/ui5-webcomponents/issues/8079

### CODE REVIEW

File: packages/fiori/src/themes/base/Bar-parameters.css [hunk 1]
```diff
@@ -4,4 +4,5 @@
 	--_ui5_bar-start-container-padding-start: 0.75rem;
 	--_ui5_bar-mid-container-padding-start-end: 0.5rem;
 	--_ui5_bar-end-container-padding-end: 0.75rem;
+	--_ui5_bar_subheader_margin-top: -1px;
```
Comment (hinzzx) [inline packages/fiori/src/themes/base/Bar-parameters.css:7]: lets just make it -0.0625rem (resulting in 1px) for consistency.

**Review by hinzzx — APPROVED** [review]
Looks good to me.

**Review by unazko — APPROVED** [review]

**Review by unazko — CHANGES_REQUESTED** [review]
Тhere is still a line between the header and sub-header in horizon and fiori high contrast themes.

**Review by unazko — CHANGES_REQUESTED** [review]
There is still a line between the two ui5-bar instances in high contrast themes.
It seems that the top margin in those cases should be -3px instead of -2px.
BarBoxShadow

**Review by unazko — APPROVED** [review]
The issue is due to the Chrome zooming feature. As a result the box shadow width isn't consistent on particular zoom levels. Same isn't reproducible with Safari and Firefox browsers.

Works fine on all browsers without zoom.

**Pull request merged**